Enhanced host monitoring plugin ver.2 (#764)
awslabs/aws-advanced-jdbc-wrapper: docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

| [Host Monitoring Connection Plugin](./using-plugins/UsingTheHostMonitoringPlugin.md)           | `efm`                     | Aurora, RDS Multi-AZ DB Cluster | Enables enhanced host connection failure monitoring, allowing faster failure detection rates. This plugin is enabled by default.                                                                                                                                                                                                                                                                                                                                                                                       | None                                                                                                                                                                                                          |

> | [Host Monitoring Connection Plugin v2](./using-plugins/UsingTheHostMonitoringPlugin.md)        | `efm2`                    | Aurora, RDS Multi-AZ DB Cluster | Enables enhanced host connection failure monitoring, allowing faster failure detection rates. This plugin is another alternative implementation for host health status monitoring. It's compatible with `efm` plugin in terms of configuration parameters. This plugin is experimental.                                                                                                                                                                                                                                | None                                                                                                                                                                                                          || Data Cache Connection Plugin                                                                   | `dataCache`               | Any database           | Caches results from SQL queries matching the regular expression specified in the  `dataCacheTriggerCondition` configuration parameter.                                                                                                                                                                                                                                                                                                                                                                                 | None                                                                                                                                                                                                          |

Reviewer: ```suggestion
| [Host Monitoring Connection Plugin v2](./using-plugins/UsingTheHostMonitoringPlugin.md#host-monitoring-plugin-v2)        | `efm2`                    | Aurora, RDS Multi-AZ DB Cluster | Enables enhanced host connection failure monitoring, allowing faster failure detection rates. This plugin is an alternative implementation for host health status monitoring. It is functionally the same as the `efm` plugin and uses the same configuration parameters. This plugin is experimental.                                                                                                                                                                                                                                | None                                                                                                                                                                                                          |
| Data Cache Connection Plugin                                                                   | `dataCache`               | Any database           | Caches results from SQL queries matching the regular expression specified in the  `dataCacheTriggerCondition` configuration parameter.                                                                                                                                                                                                                                                                                                                                                                                 | None                                                                                                                                                                                                          |
```
